src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -1738,10 +1738,26 @@ public override void VisitDeclarationPattern(IDeclarationPatternOperation operat
         {
             LogString(nameof(IDeclarationPatternOperation));
             LogSymbol(operation.DeclaredSymbol, " (Declared Symbol");
+            LogConstant((object)operation.MatchesNull, ", AcceptsNull");
```

Comment:
>AcceptsNull [](start = 58, length = 11)

Consider using `nameof` so that this stays in sync, as it's already out of sync.